apps/web/src/components/PoolCandidatesTable/PoolCandidatesTable.tsx

```diff
@@ -340,6 +343,7 @@ const defaultState = {
   hiddenColumnIds: ["candidacyStatus", "notes"],
   filters: {
     applicantFilter: {
+      publishingGroups: [PublishingGroup.ItJobs, PublishingGroup.ItJobsOngoing],
```

Comment:
I'm not 100% sold on this part
When you load up an IT pool like the CMO Digital Careers test pool this default filter changes nothing
But when you load up something else, like the IAP pool, you need to clear the default publishing group every time to have results

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/7161c8c3-2343-4feb-a647-1798595afcf9)

Reply:
Yeah, good call. I'll ask about this in standup today 👌